pom.xml

```diff
@@ -93,6 +93,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>durable-task</artifactId>
+      <version>1.39</version>
```

Comment:
Why do you need to bump this? It looks unrelated to the change.

Reply:
maven fail to build the hpi due to dependencies. if I do not bump it.
I can try to check which one is the one that allows hpi to be build.
I'll re-check with what is on master right now. and see if it build but I think I did a pull and that is where started to fail.

Reply:
I think that this is required because workflow-durable-task-step released a new version 2.40 that requires durable-task >1.36
so possible solutions are.
bump up durable tasks to 1.36 and its dependencies.
or limit  workflow-durable-task-step to 2.39
Both solutions are ok.
Which one do you prefer me to do?

Reply:
You shouldn't have to do this. Please revert the jenkins core update and this line.

Most plugin versions are managed by the jenkins bom. In general it shouldn't be changed unless there is a need for a new api or a bugfix that is affecting the plugin.

Reply:
I'm missing something. if I leave pom.xml as it is I've got this error.
`Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for org.jenkins-ci.plugins:durable-task:1.35 paths to dependency are:
+-org.csanchez.jenkins.plugins:kubernetes:1.30.3-SNAPSHOT
  +-org.jenkins-ci.plugins:durable-task:1.35
and
+-org.csanchez.jenkins.plugins:kubernetes:1.30.3-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-durable-task-step:2.40
    +-org.jenkins-ci.plugins:durable-task:1.37 (managed) <-- org.jenkins-ci.plugins:durable-task:1.36
]`

Reply:
That is weird, I don't have the same problem.
> +-org.csanchez.jenkins.plugins:kubernetes:1.30.3-SNAPSHOT
  +-org.jenkins-ci.plugins:durable-task:1.35

indicates you have a direct declaration of durable-task:1.35 in the pom.xml. Are you sure you don't have anything else staged?

Just in case, make sure you're running maven latest version, could be a bug with an ancient maven release.

Reply:
I downloaded the last pom.xml from master branch and that works.